deployer.sh

```diff
@@ -117,14 +117,13 @@ if [[ $VERBOSE != 0 ]] ; then
     echo "VERBOSE=$VERBOSE"
     echo "QUIET=$QUIET"
     echo "USAGE=$USAGE"
-fi
-
 
-if [[ ! -d ./node_modules || ! -x $KNOWN_GOOD_QX_CMD ]]; then
-    info "Running npm install in deployment"
-    npm install
+    echo node version: $(node --version)
+    echo npm version:  $(npm --version)
+    echo bootstrap qx: $KNOWN_GOOD_QX_CMD
```

Comment:
Since the variable will always be "npx qx", wouldn't it be more interesting to display the version `$($KNOWN_GOOD_QX_CMD --version)` of the qx command?